faith_presentation/src/commonMain/kotlin/net/thechance/mena/faith/presentation/feature/quran/tilwah/DownloadedReciterScreen.kt

```diff
@@ -46,20 +61,47 @@ private fun Content(
                 onSearchClick = listener::onSearchClick,
                 onBackClick = listener::onBackClick
             )
-        }) {
+        },
+        snakeBar = {
+            FaithSnackBar(
+                message = snackBar.message,
+                isVisible = snackBar.isVisible,
+                status = snackBar.status,
+            )
+        },
+        overlays = {
+            dialog(
+                isVisible = uiState.showDeleteConfirmationDialog,
+            ) {
+                DeleteConfirmationDialog(
+                    title = stringResource(Res.string.delete_reciter),
+                    message = stringResource(Res.string.delete_reciter_dialog_message),
+                    showDialog = uiState.showDeleteConfirmationDialog,
+                    onDeleteClick = listener::onConfirmDeleteReciterClick,
+                    onDismiss = listener::onDismissDeleteConfirmationDialog,
+                )
+            }
+        }
+    ) {
         LazyColumn(
             contentPadding = PaddingValues(bottom = Theme.spacing._16),
         ) {
             items(uiState.reciters) { reciter ->
                 ReciterItem(
-                    reciter = reciter.name,
-                    recitingType = reciter.recitingType,
-                    isDownloaded = reciter.isDownloaded,
+                    reciterDetails = reciter,
```

Comment:
```suggestion
                    reciterDetails = reciterDetails,
```
as it wasn't specified what related to the reciter, what do u think